jcommune-service/src/main/java/org/jtalks/jcommune/service/transactional/TransactionalTopicModificationService.java

```diff
@@ -434,4 +416,11 @@ public void openTopic(Topic topic) {
         topic.setClosed(false);
         dao.saveOrUpdate(topic);
     }
+
+    private void enableSubscriptionWithAutoSubscribe(Topic topic, JCUser currentUser){
```

Comment:
This method name looks much more weird than ```subscribeOnTopicIfNotificationsEnabled``` as for me. In fact, we don't enable subscribtion here, we subscribe user to topic. Also, I thenk, it will be mush better to keep all subscribe/unsubscribe logic in single place (SubscriptionService). So, I'd avice to use ```SubscriptionService#toggleSubscription(SubscriptionAwareEntity entityToSubscribe)``` method here. Or introduce new ```SubscriptionService#usbscribe(SubscriptionAwareEntity entityToSubscribe)```